### PR TITLE
added rspec + dependencies

### PR DESCRIPTION
**Pull request by fxckdead**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -31,6 +31,13 @@ gem 'bootsnap', '>= 1.4.2', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+
+  # Rspec
+  %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
```
Comment (groeneman) [inline Gemfile:36]: Why load directly from github as opposed to the normal syntax?

Comment (fxckdead) [reply Gemfile:36]: coz was the way they  [have it on the docs](https://github.com/rspec/rspec-rails#installation) haha, looks weird, but probably is better the other way? more readable ?

Comment (groeneman) [reply Gemfile:36]: That instruction is to run a pre-released version. I think you just want `gem :rspec-rails`

Comment (fxckdead) [reply Gemfile:36]: u right, i will use the other style then, good one ! 👍

**Pull request merged**